src/components/footer-system/README.md

```diff
@@ -1,26 +1,254 @@
+                        <path d="M17.3999 16.0002H20.9999V17.6002C21.4999 16.7002 22.6999 15.7002 24.4999 15.7002C28.2999 15.7002 28.9999 18.2002 28.9999 21.4002V28.0002H25.2999V22.2002C25.2999 20.8002 25.2999 19.0002 23.3999 19.0002C21.4999 19.0002 21.1999 20.5002 21.1999 22.1002V28.0002H17.4999V16.0002H17.3999Z" fill="#F7F7F8"></path>
+                    </svg>
+                </a>
+            </li>
+
+            <!-- Linked social media icon -->
+
+            <li>
+                <a href="https://twitter.com/IndianaUniv">
+                    <span class="rvt-sr-only rvt-color-white">Twitter for IU</span>
+                    <svg xmlns="http://www.w3.org/2000/svg" fill="none" height="40" viewBox="0 0 40 40" width="40">
+                        <path d="M20 40C31.0457 40 40 31.0457 40 20C40 8.9543 31.0457 0 20 0C8.9543 0 0 8.9543 0 20C0 31.0457 8.9543 40 20 40Z" fill="#7A1705"></path>
+                        <path d="M30.0002 13.7998C29.3002 14.0998 28.5002 14.2998 27.6002 14.3998C28.4002 13.8998 29.1002 13.0998 29.4002 12.0998C28.6002 12.5998 27.7002 12.8998 26.8002 13.0998C26.1002 12.2998 25.0002 11.7998 23.8002 11.7998C21.5002 11.7998 19.7002 13.5998 19.7002 15.8998C19.7002 16.1998 19.7002 16.4998 19.8002 16.7998C16.4002 16.5998 13.4002 14.9998 11.3002 12.4998C10.9002 13.0998 10.7002 13.7998 10.7002 14.5998C10.7002 15.9998 11.4002 17.2998 12.5002 17.9998C11.8002 17.9998 11.2002 17.7998 10.6002 17.4998C10.6002 17.4998 10.6002 17.4998 10.6002 17.5998C10.6002 19.5998 12.0002 21.1998 13.9002 21.5998C13.6002 21.6998 13.2002 21.6998 12.8002 21.6998C12.5002 21.6998 12.3002 21.6998 12.0002 21.5998C12.5002 23.1998 14.0002 24.3998 15.8002 24.3998C14.4002 25.4998 12.6002 26.1998 10.7002 26.1998C10.4002 26.1998 10.0002 26.1998 9.7002 26.0998C11.5002 27.2998 13.7002 27.8998 16.0002 27.8998C23.5002 27.8998 27.7002 21.5998 27.7002 16.1998C27.7002 15.9998 27.7002 15.7998 27.7002 15.6998C28.8002 15.2998 29.4002 14.5998 30.0002 13.7998Z" fill="#F7F7F8"></path>
+                    </svg>
+                </a>
+            </li>
+        </ul>
+    </div>
 </div>
+
+<!-- Footer resources module from previous section -->
+
+<div aria-labelledby="resources-heading" class="rvt-footer-resources" role="complementary">
+  <!-- ... -->
+</div>
+
+<!-- Base footer from previous section -->
+
 <footer class="rvt-footer-base">
-  <!-- Base module content -->
+  <!-- ... -->
 </footer>
 ```
 
-## Usability guidance
-> TODO: Write "Do this, Don't do this..." documentation
\ No newline at end of file
+## Accessibility
+
+The footer is built to follow the WAI-ARIA authoring standards. It is marked up with the appropriate ARIA attributes, described below.
+
+### ARIA attributes
+
+|Attribute|Description|
+|-|-|
+|`role="complementary"`|Applied to the container `div` for the optional resources and social modules|
```

Comment:
You've written it further up in the docs and code examples, but it may be worth mentioning here in the description of the complementary role, that the elements that use this role must be siblings of the `<main>` element. Eg. NOT nested inside it, or inside the `<footer>` (base module) element. Just to reiterate that point home.

Reply:
Can do!